### DISCUSSION TITLE
Welcome to findex Discussions!

### DISCUSSION THREAD

**Discussion by mdgaziur** [mdgaziur/findex#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GijsvM** [comment]
Hey, is there maybe a particular key you can press to start findex immediatly? i think that would be really cool.

**Reply by mdgaziur** [reply]
Currently, you have to set the shortcut key from your Desktop Environment settings. My plan is to rewrite Findex and then add new features. However, currently I'm busy studying for an upcoming exam that'll start at 19 June and finish at 20 July. I'll start working on Findex after 20 July
